spacy/displacy/__init__.py

```diff
@@ -29,8 +29,13 @@ def render(docs, style='dep', page=False, minify=False, jupyter=IS_JUPYTER,
                  'ent': (EntityRenderer, parse_ents)}
     if style not in factories:
         raise ValueError(Errors.E087.format(style=style))
-    if isinstance(docs, Doc) or isinstance(docs, dict):
+    if isinstance(docs, (Doc, dict)):
         docs = [docs]
+    elif isinstance(docs, Span):
```

Comment:
I think one case we're not covering here yet is if you pass in a sequence of `Span` objects, or maybe even a mix of `Span`s and `Doc`s? Maybe something like this could work:

```diff
- elif isinstance(docs, Span):
-     docs = [docs.as_doc()]
+ docs = [obj for obj in docs if not isinstance(obj, Span) else obj.as_doc()]
```

So basically, by that point, we can assume that `docs` is in fact a list / iterable, right? And if a member of that list is an instance of `Span`, we convert it to a `Doc`.